setup.py

```diff
@@ -9,8 +9,8 @@
     author="",
     author_email="",
     url="https://github.com/user/project",
-    install_requires=["lightning", "hydra-core", "hydra-colorlog"],
-    packages=find_packages(),
+    # install_requires=["lightning", "hydra-core", "hydra-colorlog"],
+    packages=find_packages(include=['tcn_hpl', 'tcn_hpl.*']),
```

Comment:
LGTM if this works for our purpose, was wondering if a more compact specification was equivalently functional
```suggestion
    packages=find_packages(include=['tcn_hpl.*']),
```